styles/styles.css

```diff
@@ -428,48 +418,7 @@ a:hover {
   }
 }
 
-/* ** Images ** */
-
-main img {
-  max-width: 100%;
-  width: auto;
-  height: auto;
-}
```

Comment:
I'm wondering if some of this image stuff such as the `max-width: 100%` might still be important for handling big images added within the content.

Reply:
we've got another ticket for the image work so I'm going to say that'll be handled later and less globally